src/Store.h

```diff
@@ -115,10 +115,7 @@ class StoreEntry : public hash_link, public Packable
     int validToSend() const;
     bool memoryCachable(); ///< checkCachable() and can be cached in memory
 
-    /// if needed, initialize mem_obj member w/o URI-related information
-    MemObject *makeMemObject();
-
-    /// initialize mem_obj member (if needed) and supply URI-related info
+    /// initialize mem_obj (if needed)
```

Comment:
If the sketch described in another change request is implemented, then this documentation will need to be adjusted further. Please check other affected methods as well.

Reply:
Done.